lib/commands/config.js

```diff
@@ -73,9 +81,25 @@ const publicVar = k => {
       }
     }
   }
+  // Redacted fields are public unless they contain redacted info
+  if (redacted.includes(k)) {
+    return redact(v) === v
+  }
   return true
 }
 
+const displayVar = (k, v) => {
+  let value
+  if (publicVar(k, v)) {
+    value = JSON.stringify(v)
+  } else if (redacted.includes(k)) {
```

Comment:
We've already determined within `publicVar` that if it returns `true` then `redact(v) === v`, but I think it does make the code clearer by eliminating an if branch. So it would look like:

```js
const displayVar = (k, v) => {
  let value
  if (publicVar(k, v) || redacted.includes(k)) {
    value = JSON.stringify(redact(v))
  } else {
    value = '(protected)'
  }
  return `${k} = ${value}`
}
```

Reply:
`publicVar` already looks in `redacted` and only returns true if it's not needed you don't need to check `redacted` again.

Reply:
But we want to display `redacted` differently than `protected`, and both states are a subset of non-public.